### ISSUE TITLE
BUG: users visible when signed out

### ISSUE THREAD

**Issue by larssondenise** [alkem-io/client-web#4114]
**Describe the bug**
when I am signed out, I cannot see any of the users. Howver, when I dive into one of the public spaces I can see all profiles and contributions. this seems inconsistent?

**To Reproduce**

**Expected behavior**

**Screenshots**
![image.png](https://images.zenhubusercontent.com/6200e10a5561c893d9e84591/f64009ae-e449-4d9e-98aa-e4b79565333d)

**Comment by Comoque1** [comment]
@larssondenise, you are right, the users, should not be visible. Could you please provide visual and expected behavior about it - how it should look like on the different blocks (i.e. post callout, activity logs on community page, etc..) ?
